app/Http/Controllers/WikiEntityImportController.php

```diff
@@ -30,7 +30,7 @@ public function create(Request $request): \Illuminate\Http\JsonResponse
     {
         $validatedInput = $request->validate([
             'wiki' => ['required', 'integer'],
-            'source_wiki_url' => ['required', 'string'],
+            'source_wiki_url' => ['required', 'url'],
```

Comment:
This is a drive by fix that prevents a potential code injection vector. The value is passed to the job's command, so it's important this does not contain potential code snippets we don't want to run. A well formed URL should be safe from what I understand.

Reply:
good catch!